presto-main/src/main/java/com/facebook/presto/tracing/TracingConfig.java

```diff
@@ -24,6 +24,7 @@
     {
         public static final String NOOP = "noop";
         public static final String SIMPLE = "simple";
+        public static final String OTEL = "otel";
```

Comment:
@tanjialiang I'm not sure if it's ok to share the artillery tracer's logic. But might be good to demonstrate some code from presto-facebook to show we could we add tracers dynamically.

Reply:
You mean adding some example code/documentations to demonstrate an addition of a tracer?